requirements/edx/development.txt

```diff
@@ -42,6 +42,7 @@ git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.1.6#egg=xblock-d
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@138e6fa0bf3a2013e904a085b9fed77dab7f3f21#egg=xblock-google-drive
 git+https://github.com/open-craft/xblock-poll@add89e14558c30f3c8dc7431e5cd6536fff6d941#egg=xblock-poll==1.5.1
 -e common/lib/xmodule
+-e git+https://github.com/edx/edx-val.git@noraiz/EDUCATOR-3566#egg=edxval
```

Comment:
This is just for testing purpose. Once Changes (https://github.com/edx/edx-val/pull/165/files) are approved, will be drafting and using a release tag.